SurveyAppTests/Tests/Services/Network/API/UserSessionTests.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  UserSessionTests.swift
+//  SurveyAppTests
+//
+//  Created by Chananchida F. on 6/17/21.
+//
+
+import Quick
+import Nimble
+
+@testable import SurveyApp
+
+class UserSessionTests: QuickSpec {
+    
+    override func spec() {
+        
+        var keychain: KeychainStorageProtocol!
+        var userSession: UserSession!
+        
+        let sampleUserCredential: UserCredential = JSON.AuthenticationService.sampleUserCredential.decoded()
+    
+        describe("Test UserSession components") {
+            
+            beforeEach {
+                keychain = KeychainStorage.default
+                userSession = UserSession(keychain: keychain)
+                UserSessionProvider.shared.userSession = userSession
+                userSession.userCredential = sampleUserCredential
+            }
+            
+            afterEach {
+                UserSessionProvider.shared.clearCredentials()
+            }
+            
+            context("when setting user credentials") {
+                it("should set the access and refresh token in UserSession") {
+                    expect(userSession.userCredential?.accessToken) == sampleUserCredential.accessToken
+                    expect(userSession.userCredential?.refreshToken) == sampleUserCredential.refreshToken
+                }
+                it("should be logged in") {
+                    expect(UserSessionProvider.shared.isLoggedIn) == true
+                }
```

Comment:
Please follow convention https://nimblehq.co/compass/development/code-conventions/quick-nimble as much as possible. 

```suggestion
                it("should set the access token in UserSession") {
                    expect(userSession.userCredential?.accessToken) == sampleUserCredential.accessToken
                }
                
                
                it("should set the refresh token in UserSession") {
                    expect(userSession.userCredential?.refreshToken) == sampleUserCredential.refreshToken
                }
                
                it("should be logged in") {
                    expect(UserSessionProvider.shared.isLoggedIn) == true
                }
```

Reply:
@blyscuit I updated the code please help check again 🙏